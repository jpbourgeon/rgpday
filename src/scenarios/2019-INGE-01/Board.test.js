/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'
import BoardComponent from './Board'
import jssSerializer from 'src/utils/jssSnapshotSerializer'

expect.addSnapshotSerializer(jssSerializer)

describe('The 2019-INGE-01 Board component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<BoardComponent team={{ name: 'hello world' }} services={[]} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
