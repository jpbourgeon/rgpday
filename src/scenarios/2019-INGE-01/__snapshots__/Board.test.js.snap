// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The 2019-INGE-01 Board component should render correctly 1`] = `
<svg>
  board.svg
</svg>
`;
